ModelBundle/DataFixtures/MongoDB/DemoContent/HomeRootDataGenerator.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
+namespace OpenOrchestra\ModelBundle\DataFixtures\MongoDB\DemoContent;
+
+use OpenOrchestra\ModelBundle\Document\Block;
+use OpenOrchestra\ModelBundle\Document\Node;
+use OpenOrchestra\DisplayBundle\DisplayBlock\Strategies\TinyMCEWysiwygStrategy;
+use OpenOrchestra\ModelInterface\Model\NodeInterface;
+
+/**
+ * Class HomeRootDataGenerator
+ */
+class HomeRootDataGenerator extends AbstractDataGenerator
```

Comment:
name class is weird and not very explicit why not call HomefunctionDataGenerator ?